packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -2151,9 +2151,10 @@ class Navigator extends StatefulWidget {
   /// This is the default value of [onGenerateInitialRoutes], which is used if
   /// [initialRoute] is not null.
   ///
-  /// If this string contains any `/` characters, then the string is split on
-  /// those characters and substrings from the start of the string up to each
-  /// such character are, in turn, used as routes to push.
+  /// If this string starts with a `/` character and has multiple `/` characters
+  /// in it, then the string is split on those characters and substrings from
+  /// the start of the string up to each such character are, in turn, used as
+  /// routes to push.
```

Comment:
This long sentence was a little confusing. Does this mean each substring split by `/` is treated as an individual route?

Reply:
The substrings from the start of the string up to each `/`is treated as individual routes. The next paragraph describes an example. I am not sure to better describe it.

Reply:
Ok! SGTM